app/views/components/gds/gds_main_template.scala.html

```diff
@@ -30,7 +30,7 @@
 
 @(  title: Title,
     backButtonTitle: String = "Back",
-    backButtonUrl: String = "/")(contentBlock: Html)(implicit request: Request[_], messages: Messages)
+    backButtonUrl: String = "/", requiresBackLink: Boolean = true)(contentBlock: Html)(implicit request: Request[_], messages: Messages)
```

Comment:
This line looks like not being formatted.

Reply:
What do you mean?  I try not to format .html pages as they seem to get completely messed up

Reply:
I mean `requiresBackLink: Boolean = true` would do better in a new line.
And I also don't format automatically.

Reply:
Gotcha!  Thanks